src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -212,18 +212,6 @@ module sonic-device_metadata {
                                  By default SONiC advertises /128 subnet prefix in Loopback0 as /64 subnet route";
                 }
 
-                leaf suppress-fib-pending {
```

Comment:
i suggest to keep these yang model instead of simple reverting.

Reply:
@lguohan I don't want to keep CLI and YANG model for a feature that does not work in a release